sys/include/phydat.h

```diff
@@ -138,6 +138,16 @@ typedef struct {
     int8_t scale;               /**< the scale factor, 10^*scale* */
 } phydat_t;
 
+/**
+ * @brief   Minimum value for phydat_t::val
+ */
+#define PHYDAT_MIN  INT16_MIN
```

Comment:
Parenthesis around `INT16_MIN`. This is not only a style issue but it might be defined in some `libc` with operators, so using `PHYDAT_MIN` might yield some weird errors when using this `libc`

Reply:
addressed